src/plugins/discover/public/components/discover_grid/render_custom_toolbar.scss

```diff
@@ -0,0 +1,66 @@
+.dscGridToolbar {
+  padding: $euiSizeS;
+  border-bottom: $euiBorderThin;
+}
+
+.dscGridToolbarControlButton .euiDataGrid__controlBtn {
+  block-size: $euiSizeXL;
+  border: $euiBorderThin;
+  border-radius: $euiBorderRadiusSmall;
+
+  // making the icons larger than the default size
+  & svg {
+    inline-size: $euiSize;
+    block-size: $euiSize;
+  }
+
+  // cancel default background and font changes
+  &.euiDataGrid__controlBtn--active {
+    font-weight: $euiFontWeightMedium;
+  }
+  &:active, &:focus {
+    background: transparent;
+  }
+
+  // add toolbar control animation
+  transition: transform $euiAnimSpeedNormal ease-in-out;
+  &:hover {
+    transform: translateY(-1px);
+  }
+  &:active {
+    transform: translateY(0);
+  }
+}
+
+.dscGridToolbarControlGroup {
+  border: $euiBorderThin;
+  border-radius: $euiBorderRadiusSmall;
+  display: inline-flex;
+  align-items: stretch;
+  flex-direction: row;
+}
+
+.dscGridToolbarControlIconButton .euiButtonIcon {
+  inline-size: $euiSizeXL;
+  block-size: $euiSizeXL;
+
+  // cancel default behaviour
+  &:hover, &:active, &:focus {
+    background: transparent;
+    animation: none !important;
+    transform: none !important;
+  }
+
+  .dscGridToolbarControlIconButton + & {
+    border-inline-start: $euiBorderThin;
+    border-radius: 0;
+  }
+}
+
+.dscGridToolbar .euiTabs {
+  transform: translateY(9px);
```

Comment:
Reduce to 8px and use EUI variable.
```suggestion
  transform: translateY($euiSize);
```

Reply:
@andreadelrio If we change to 8 then the blue border would be further from the gray border:

Documents tab:
<img width="229" alt="Screenshot 2023-10-18 at 14 59 58" src="https://github.com/elastic/kibana/assets/1415710/31f61170-ec0a-4e02-97e1-448a8a514068">

vs Statistics tab:
<img width="231" alt="Screenshot 2023-10-18 at 15 00 07" src="https://github.com/elastic/kibana/assets/1415710/f689c3be-b7de-4910-a1ec-ff0ae842a4fe">

`9px` seems to be working better to align both tabs borders visually.